### PR TITLE
Update README and package.json

### PR DESCRIPTION
**Pull request by DominicSherman**
* Making these files cardwallet specific
* Updating docs
* Adding documentation on syncing environment variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,79 @@
...
 
+> DeFi payments made easy
 ## Requirements
 
 * A computer running macOS.
 * NVM installed or Node.js 14: https://github.com/creationix/nvm
 * Install CocoaPods by running `sudo gem install cocoapods`
 * Install Watchman `brew install watchman`
-* Install the latest version of XCode: https://developer.apple.com/xcode/
+* xCode Version 12.4 (can be found [here](https://developer.apple.com/download/all/?q=xcode))
```
Comment (danibonilha) [inline README.md:10]: is https://bundler.io/ also a requirement ?

File: README.md [hunk 2]
```diff
@@ -1,73 +1,79 @@
...
+CRYPTEX_PASSWORD=${{CRYPTEX_PASSWORD}}
+CRYPTEX_SKIP_DOCS=true
+ANDROID_KEYSTORE_ALIAS=android_release_keystore
+ANDROID_KEYSTORE_PATH=./fastlane/android/signing/release.keystore
+ANDROID_KEYSTORE_CRYPTEX_KEY=android_release_keystore
+MATCH_GIT_URL=https://github.com/cardstack/cardwallet-context
+MATCH_PASSWORD=${{CRYPTEX_PASSWORD}}
+MATCH_STORAGE_MODE=git
+MATCH_TYPE=appstore
+MATCH_APP_IDENTIFIER=com.cardstack.cardpay
```
Comment (danibonilha) [inline README.md:71]: Could we have those env variables as default  in the fastlane/.env and instruct to go to this file and replace the  `CRYPTEX_PASSWORD` instances ?

File: package.json [hunk 3]
```diff
@@ -457,4 +457,4 @@
       "outputFile": "./cardstack/storybook/storyLoader.js"
     }
   }
-}
+}
```
Comment (danibonilha) [inline package.json:460]: EOF

**Review by danibonilha — APPROVED** [review]
LGTM, just some suggestions

File: README.md (hunk 1 above)
Comment (DominicSherman) [reply README.md:10]: I don't think it is for this app, but we could make it a requirement. I don't think I've _had_ to run it any point

File: README.md (hunk 2 above)
Comment (DominicSherman) [reply README.md:71]: We could do an `.env.example` file with these preset but you'd still have to create a `.env` file because we need to gitignore it

**Review by lukemelia — APPROVED** [review]
📈

**Pull request merged**